os/arch/arm/src/armv7-r/arm_mpuguardrestore.S

```diff
@@ -0,0 +1,53 @@
+/****************************************************************************
+ *
+ * Copyright 2016 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+#include <tinyara/irq.h>
+#include <chip/mpu-reg.h>
+#include "up_internal.h"
+#include "svcall.h"
+#include "mpu.h"
+
+	.file	"arm_mpuguardrestore.S"
+
+	.globl  up_mpuguardrestore	
```

Comment:
Shouldn't it be .global?

Reply:
Anway, Both spellings (`.globl' and `.global') are accepted. Only thing is we need to maintain uniformity. 
No problem, lets keep .globl only.!!!!